### PR TITLE
Merge branch 'master' into next

### PR DESCRIPTION
**Pull request by cemremengu**
We can can squash merge it from here, I can directly push or any other way that you suggest.

### CODE REVIEW

File: lib/contentTypeParser.js [hunk 1]
```diff
@@ -152,7 +152,9 @@ function rawBody (request, reply, options, parser, done) {
     req.removeListener('error', onEnd)
 
     if (err !== undefined) {
-      reply.code(400).send(err)
+      const error = new Error(err.message)
+      error.statusCode = 400
+      reply.code(error.statusCode).send(error)
```
Comment (mcollina) [inline lib/contentTypeParser.js]: This is losing the stacktrace, it should probably have not landed as is.

**Comment by cemremengu** [comment]
What the???

**Comment by cemremengu** [comment]
I didn't intend to merge this (without squashing at least) but somehow I managed to screw up and there is no way back. I apologize...

I guess we should do a soft reset and force push?

**Comment by mcollina** [comment]
Go for it.

**Comment by mcollina** [comment]
We'll have to land these manually.

**Comment by cemremengu** [comment]
I landed your suggestion actually but pushed here instead of my branch...

**Comment by mcollina** [comment]
I'll fix this and force push on next.

**Comment by cemremengu** [comment]
Thank you, I apologize again

**Comment by mcollina** [comment]
You'll have to redo your change (do them against master)

**Comment by cemremengu** [comment]
You mean merge master to next again?

**Comment by cemremengu** [comment]
Oh ok I understand now.

**Comment by cemremengu** [comment]
Do you usually squash when merging to next or you didn't for this time since I messed up?

**Comment by cemremengu** [comment]
I hope it's ok now, sorry for wasting your time. I promise I won't touch this kind of things again for a long time :)

**Pull request merged**